packages/application-shell/src/test-utils/test-utils.js

```diff
@@ -268,7 +268,7 @@ const renderApp = (
     permissions, // <-- deprecated option, use `{ project: { allAppliedPermissions } }`
     actionRights, // <-- deprecated option, use `{ project: { allAppliedActionRights } }`
     dataFences, // <-- deprecated option, use `{ project: { allAppliedDataFences } }`
-    dataLocale = 'en',
```

Comment:
Ah good catch. Maybe we can add `en` to the project languages list instead.

Reply:
I think that way it's simpler. will do that.